beep/features/featurizer_helpers.py

```diff
@@ -370,11 +370,16 @@ def get_hppc_ocv(processed_cycler_run, diag_pos):
     cycle_hppc_0 = cycle_hppc.loc[cycle_hppc.cycle_index == cycles[0]]
     #     in case that cycle 2 correspond to two cycles one is real cycle 2, one is at the end
     cycle_hppc_0 = cycle_hppc_0.loc[cycle_hppc_0.test_time < 250000]
-
-    step = 11
-    step_later = 43
-
-    voltage_1 = get_hppc_ocv_helper(cycle_hppc_0, step)
+    first_diagnostic_steps = get_step_index(processed_cycler_run,
+                                            cycle_type="hppc",
+                                            diag_pos=0)
+    later_diagnostic_steps = get_step_index(processed_cycler_run,
+                                            cycle_type="hppc",
+                                            diag_pos=0)
```

Comment:
Should `diag_pos` be the method arg `diag_pos` for `later_diagnostic_steps` ?

Reply:
yes, I think that's the correct usage here